css-coding-conventions.md

```diff
@@ -0,0 +1,7 @@
+# CSS Coding Conventions
+
+## Introduction 
+This document is written to capture the conventions of writing styles for our projects. Many of the rules will be or are part of our linter- and prettier settings. Due to variety of our projects and methods there might be a need for a more detailed approach per situation. This might also impact how code is being organized, structured on an architectonic level. Therefore, this part of the conventions will focus on the rules that are applicable to the majority of our implementation methods.
+
+## Disclaimer
+It's important to note that Client- and project requirements always undo choices made in these guidelines
```

Comment:
_Client_ shouldn't be capitalized? unless we are treating it the same as **God** :sweat_smile: 
Also not sure why it is followed by dash, same dash appears on line 4 after _linter_

Reply:
Perhaps it's dunglish. But AFAIK it is the way of writing:
`client requirements and project requirements` in a short way.